tests/ingest_agents.py

```diff
@@ -21,14 +21,18 @@ def __init__(self, deployment):
         self.ingest_auth_agent = IngestAuthAgent()
         self.auth_headers = self.ingest_auth_agent.make_auth_header()
 
-    def upload(self, metadata_spreadsheet_path, is_update=False, project_uuid=None):
+    def upload(self, metadata_spreadsheet_path, is_update=False, project_uuid=None, submission_uuid=None):
         url = self.ingest_broker_url + '/api_upload'
         if is_update:
             url = self.ingest_broker_url + '/api_upload_update'
 
         data = {}
         if project_uuid:
             data['projectUuid'] = project_uuid
+
+        if submission_uuid:
```

Comment:
What happens if `submission_uuid` is `None`?

Reply:
Just not passed in the post data. Case seems to be covered in broker SpreadsheetUploadService.async_upload (project and sub uuid can default to None).

Reply:
Yeah, but if the submission UUID is none, it's going to create a new submission, it shouldn't for updates.